File: 4 June/Solution.java
class Solution {
    public void reverseString(char[] s) {
        
        int n=s.length-1;
        int i=0;
        
        reverse(s,i,n);
        System.out.println(s);
        
    }
    
    public void reverse(char []s,int i , int n)
    {
        char t='\0';
        if(i<n)
        {
            t=s[i];
            s[i]=s[n];
            s[n]=t;
            reverse(s,i+1,n-1);
        }
    }
}

//Using two pointer approach. 

File: 4 June/Solution.c
void reverseString(char* s, int sSize){

    int i =0;
    int j=sSize-1;
    char t='\0';
    for(i=0;i<j;i++)
    {
        t=s[i];
        s[i]=s[j];
        s[j]=t;
        j--;
    }
}